### ISSUE TITLE
Clean Url → Base64 encrypt url HELP ME

### ISSUE THREAD

**Issue by Tenakskd** [nfriedly/nodeunblocker.com#48]
In my school,
I can't use it with a blocked url after the /proxy/. So how can I hide the url with base64 encoding and encrypt it?

**Comment by nfriedly** [comment]
Yeah, there's a ticket for this at https://github.com/nfriedly/node-unblocker/issues/141 - I've looked into it, but it's more work than it sounds at first, because a lot of what makes this proxy special is how it handles URLs, so base64 encoding them would require re-writing quite a lot of the proxy.
